funnel/jobs/jobs.py

```diff
@@ -43,7 +45,18 @@ def tag_locations(project_id):
                 return
             url = urljoin(app.config['HASCORE_SERVER'], '/1/geo/parse_locations')
             response = requests.get(
-                url, params={'q': project.location, 'bias': ['IN', 'US']}
+                url,
+                params={
+                    'q': project.location,
+                    'bias': ['IN', 'US'],
+                    'special': [
+                        __('Anywhere'),
+                        __('Home'),
+                        __('Internet'),
+                        __('Online'),
+                        __('Remote'),
```

Comment:
Since "Home", "Anywhere" and "Remote" are unlikely for location, keep just "Online" and "Internet".

Also, Online and Internet need non-localized English language versions in addition to localized versions since there is a high chance of a Hindi user typing "Online" instead of the Hindi word. That makes the list:

`['Online', 'Internet', __("Online"), __("Internet")]`